pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -89,6 +91,21 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 	if !ok {
 		return nil, false, errors.NewBadRequest(fmt.Sprintf("wrong object passed to Scale update: %v", obj))
 	}
+
```

Comment:
Fair enough.  The main thing it brought was consistency with the other validators, but this is kind of a weird case since there's no "create" action for scale anyway.